misc/python/materialize/checks/actions.py

```diff
@@ -40,7 +40,10 @@ def execute(self, c: Composition) -> None:
         c.sql(
             """
             DROP CLUSTER REPLICA default.default_replica;
-            CREATE CLUSTER REPLICA default.default_replica REMOTE ['computed_1:2100'];
+            CREATE CLUSTER REPLICA default.default_replica
+                REMOTE ['computed_1:2100'],
```

Comment:
What is the difference between `REMOTE` and `COMPUTE` ?

Reply:
*  REMOTE says how environmentd connects to computed processes
* COMPUTE specifies to which address/port the timely connections are set up

If there is only one computed process, COMPUTE can be empty